admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -1,15 +1,18 @@
 package io.buoyant.admin
 
 import io.buoyant.config.types.Port
-import java.net.InetSocketAddress
+import java.net.{InetAddress, InetSocketAddress}
 
 case class AdminConfig(
+  ip: Option[InetAddress],
   port: Option[Port],
```

Comment:
Yeah, for backward compatibility we either need to get rid of `AdminConfig` class and replace it with `HostAndPort` or stick to current implementation.

IMHO, it might be worth sticking to current implementation as `AdminConfig` might need extra config parameters later. And we always can refactor it and do what you suggested.

Please let me know what you think and what should I do.